state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -252,14 +271,15 @@ pub mod pallet {
 	#[derive(CloneNoBound, RuntimeDebug, PartialEq, Eq, Encode, Decode, TypeInfo)]
 	#[scale_info(skip_type_params(T, I))]
 	pub struct DepositChannelDetails<T: Config<I>, I: 'static> {
+		/// The owner of the deposit channel.
+		pub owner: T::AccountId,
```

Comment:
This will need a migration. In the migration, we won't know the owner, so will just have to use some default value.